win32ss/user/winsrv/consrv/frontends/gui/conwnd.c

```diff
@@ -2452,6 +2452,9 @@ ConWndProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
 
             if (DefWindowProcW(hWnd /*GuiData->hWindow*/, WM_NCHITTEST, 0, lParam) == HTCLIENT)
             {
+                /* Don't display the context menu if QuickEdit is enabled */
+                if (Console->QuickEdit) break;
```

Comment:
But one may still want to have the menu, even in quick-mode (for e.g. searching strings in the console or do the other operations it offers). Since it becomes unavailable when right-clicking, it would be useful to get it through the other standard means. For example with that Shift-F10 you mention. In which cases, blocking the menu in the `WM_CONTEXTMENU` doesn't sound good.

Reply:
Windows does not allow you to bring up the context menu at all if QuickEdit is on. You can however reach the same menu by opening the system menu.

Reply:
Just being a neutral person here, not taking any of the sides, can you prove that by writing an API test that sends `WM_CONTEXTMENU` programatically to the console window with QuickEdit mode enabled? Then you can act depending on the experiment result.

Reply:
(Note that such a test must run as `SYSTEM` for it to have any chance to test anything, since consrv/winsrv runs under csrss.exe)